### DISCUSSION TITLE
Can't get date on FILE guessing from Guessing from date on file

### DISCUSSION THREAD

**Discussion by paristotle** [TheLastGimbus/GooglePhotosTakeoutHelper#345] [I NEED HELP] [answered]
This is my first attempt at running the takeouthelper and the output on the screen indicates Can't get date on _Path\File_ guessing from Guessing from date on file. Did I do something wrong when running the application?

**Comment by TheLastGimbus** [comment, marked as answer]
nope, it will even print out how many of them were successful. It is expected that gpth may not manage to find out everything :ok_hand:
